cli/mender-artifact/copy.go

```diff
@@ -37,7 +37,13 @@ func Cat(c *cli.Context) (err error) {
 	if !isimg.MatchString(c.Args().First()) {
 		return cli.NewExitError("The input image does not seem to be a valid image", 1)
 	}
-	r, err := virtualPartitionFile.Open(comp, c.Args().First())
+
+	privateKey, err := getKey(c.String("key"))
```

Comment:
I guess this is on me, as one of the things that should have been there in the first place :p